File: src/main/java/com/exmple/kyudo_app/domain/model/Team.java
package com.exmple.kyudo_app.domain.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
@Entity
@Table(name="Team")

public class Team {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name ="team_id")
    private Long teamId;
    @Column(name ="team_name")
    private String teamName;


}

File: src/main/kotlin/com/example/kyudo_app/infrastructure/persistence/Team.kt
package com.example.kyudo_app.infrastructure.persistence

import jakarta.persistence.*
import lombok.Getter
import lombok.Setter

@Setter
@Getter
@Entity
@Table(name = "Team")
class Team {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "team_id")
    private var teamId: Long? = null

    @Column(name = "team_name")
    private var teamName: String? = null
}
